interpreter/python/python.go

```diff
@@ -803,11 +805,20 @@ func Loader(ebpf interpreter.EbpfHandler, info *interpreter.LoaderInfo) (interpr
 		vms.PyFrameObject.Code = 0
 		vms.PyFrameObject.LastI = 56       // _Py_CODEUNIT *prev_instr
 		vms.PyFrameObject.Back = 8         // struct _PyInterpreterFrame *previous
-		vms.PyFrameObject.EntryMember = 70 // char owner
+		vms.PyFrameObject.EntryMember = 70 // struct _PyInterpreterFrame owner
```

Comment:
Is this inteded change? The member `owner` is of type `char`. The comment is intended to imply the field type, not the struct type.

Reply:
oh :see_no_evil: I was reading it incorrectly. As the referenced field is actually in the [struct _PyInterpreterFrame](https://github.com/python/cpython/blob/624a8e4f976bf64851d51ff7ff7d7e24ad87c142/Include/internal/pycore_frame.h#L61) instead of something like PyFrameObject, I thought `struct _PyInterpreterFrame` points to the new location. I will adjust it.